docs/index.html

```diff
@@ -46,7 +46,7 @@
   </script>
   <script src="https://cdn.jsdelivr.net/npm/docsify/lib/docsify.min.js"></script>
   <script src="https://cdn.jsdelivr.net/npm/docsify-themeable@0/dist/js/docsify-themeable.min.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/docsify@4/lib/plugins/search.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/docsify/lib/plugins/search.min.js"></script>
```

Comment:
I see you are pointing to the latest minified version of search.js. Are these changes in the latest version of docsify that is fixing this issue that is not there in major version 4? 
It would be good to specify a particular version so that we don't get affected by newer version updates.
https://docsify.js.org/#/quickstart?id=specifying-docsify-versions

Reply:
Reverting the above change. Closing this PR and submitting the above changes in a new PR as facing some issues in committing the new changes.